guides/model-state.md

```diff
@@ -0,0 +1,13 @@
+## Avoid Local/UI state on Models
+
+Models live on the `store` service, and all services hold "application state."
+UI or local state should not be entangled with application state by adding
+it to a model.
```

Comment:
@mike-north I strongly agree that we should try to avoid introducing terminology when we don't need to.  Doing so is a common vice that makes many things seem more complicated than they actually are.